test/unbox.test.js

```diff
@@ -36,8 +36,10 @@ test('unbox', async t => {
   }
 
   const RECPS = [
+    // TODO: i think i asked this somewhere else, but do we actually wanna support DMs in the first slot in this module? what's the usecase?
```

Comment:
made a thread about this in some other file above

Reply:
remove TODO

Reply:
removed comment and unskipped test